gen/dcos-config.yaml

```diff
@@ -211,9 +211,57 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
+  - path: /etc_master/logrotate_master.config
+    content: |
+      compress
+      delaycompress
+      notifempty
+      missingok
+      nomail
+      /var/lib/dcos/mesos/log/* {
+          olddir /var/lib/dcos/mesos/log/archive
+          maxsize 256M
+          rotate {{ mesos_log_retention_count }}
+          copytruncate
+          postrotate
+              find /var/lib/dcos/mesos/log/ /var/lib/dcos/mesos/log/archive -mtime +14 -delete
```

Comment:
With the existing `postrotate` you can create an infinite number of files in the log directory and they won't be deleted until 2 weeks later.  The `-mtime +14` is not a file count.

`find ... -mtime +14` selects all files with a modification of at least 15 days ago (according the man page):

> -mtime n
              File's data was last modified n*24 hours ago.  See the
              comments for -atime to understand how rounding affects the
              interpretation of file modification times.

And `-atime` says:
> -atime n
              File was last accessed n*24 hours ago.  When find figures out
              how many 24-hour periods ago the file was last accessed, any
              fractional part is ignored, so to match -atime +1, a file has
              to have been accessed at least two days ago.